tools/scripts/data.toml

```diff
@@ -19,13 +19,7 @@ args = [
 description = "Extracts ICU4X resource keys used by a binary."
 category = "ICU4X Data"
 script_runner = "bash"
-script = """
-    # TODO(#1106): Implement this
-    echo "datetime/lengths@1" > ${2}
-    echo "datetime/skeletons@1" >> ${2}
-    echo "datetime/symbols@1" >> ${2}
-    echo "plurals/ordinal@1" >> ${2}
-"""
+script = '''strings ${1} | grep -e "icu4x_key_tag\w" | sed s/icu4x_key_tag// | sort > ${2}'''
```

Comment:
Thought: Although using `strings` is handy, we should still offer a cross-platform solution I think. Opinions?

Reply:
I don't know how complex `strings` is, i.e. it can probably handle many different binary formats etc. Could be hard to match its sophistication.

Reply:
Did you verify manually that it works for at least x86_64 (elf) and wasm files?

Reply:
Did you verify manually that it works for at least x86_64 (elf) and wasm files?

Reply:
https://superuser.com/questions/124081/is-there-a-windows-equivalent-of-the-unix-strings-command

Reply:
I manually verified ELF, and WASM I believe as well, but that's covered by the CI test anyway.

Reply:
There's this https://docs.microsoft.com/en-us/sysinternals/downloads/strings. The piping, `grep`, and `sed` probably won't work on windows though...

Reply:
It makes me uneasy to think that a pivotal part of the ICU4X data pipeline rests on piping `strings` to `sed`. I'd rather have a tool that takes an executable and returns the key list.

(My approval stands; please file a follow up to discuss further)